### PR TITLE
feat(made-to-stick): user role based accessibility

### PR DESCRIPTION
**Pull request by samim-hossain-mondal**
- get, create, update, delete data by specific projectId
- user role based accessibility to note cards

### CODE REVIEW

File: src/controllers/madeToStick.controller.js [hunk 1]
```diff
@@ -87,20 +40,20 @@ const deleteMadeToStick = async (req, res, next) => {
   }
 };
 
-const getAllMadeToStick = async (req, res, next) => {
-  const projectId = parseInt(req.params.projectId);
-
+const deleteMadeToStick = async (req, res, next) => {
   try {
-    const madeToStickItem = await madeToStickService.getAllMadeToStick(projectId);
+    const { i } = req.params;
```
Comment (sugumar-mani) [inline src/controllers/madeToStick.controller.js:45]: what is `i`? give proper variable naming if possible

File: src/middlewares/index.js [hunk 2]
```diff
@@ -11,6 +11,8 @@ function errorHandlingMiddleware(err, req, res, next) {
     return next(err);
   }
 
+  console.log(err.message);
```
Comment (sugumar-mani) [inline src/middlewares/index.js:14]: remove if console not needed

File: src/services/dsm/celebrationBoard.services.js [hunk 3]
```diff
@@ -166,39 +164,61 @@ const updateReaction = async (celebrationId, memberId, isReacted, projectId) =>
...
+          memberId: true,
+          isRemoved: true
+        }
+      }
     }
   });
 
-  if (!celebration) throw new HttpError(404, 'No Record Found');
-  if (celebration.memberId === memberId) throw new HttpError(403, 'You are not authorized to perform this action');
+  console.log(celebration);
```
Comment (sugumar-mani) [inline src/services/dsm/celebrationBoard.services.js:179]: re,ove unnecessary logs

Comment (samim-hossain-mondal) [reply src/services/dsm/celebrationBoard.services.js:179]: removed !

File: src/controllers/madeToStick.controller.js (hunk 1 above)
Comment (samim-hossain-mondal) [reply src/controllers/madeToStick.controller.js:45]: in the db table 'i' is the attribute name given to a specific note card id

File: src/middlewares/index.js (hunk 2 above)
Comment (samim-hossain-mondal) [reply src/middlewares/index.js:14]: removed !

**Review by onegoel — APPROVED** [review]

**Pull request merged**